pkgs/by-name/e1/e1s/package.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub }:
```

```suggestion
{
  stdenv,
  lib,
  buildGoModule,
  fetchFromGitHub,
}:
```

Formatted as per RFC 0166

---

This isn't enforced as of yet, and once it's enforced would be done so by CI. Until then we don't need to nit about it.

But yes, eventually we will switch to using `nixfmt-rfc-style` treewide.